src/lualib/Spread.ts

```diff
@@ -1,7 +1,13 @@
 function __TS__Spread<T>(this: void, iterable: Iterable<T>): T[] {
-    const arr: T[] = [];
-    for (const item of iterable) {
-        arr[arr.length] = item;
+    const arr = [];
+    if (typeof iterable === "string") {
+        for (const [character] of string.gmatch(iterable, ".")) {
```

Comment:
I assume it would be significantly slower than a regular for loop. Is some reason why it wouldn't work?